manifests/m/Microsoft/PowerToys/0.41.4/Microsoft.PowerToys.installer.yaml

```
@@ -0,0 +1,16 @@
+# Created using wingetcreate
+
+PackageIdentifier: Microsoft.PowerToys
+PackageVersion: 0.41.4
+MinimumOSVersion: 10.0.17134.0
+InstallerType: exe
+InstallerSwitches:
+  Silent: --silent
+  SilentWithProgress: --silent
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/microsoft/PowerToys/releases/download/v0.41.4/PowerToysSetup-0.41.4-x64.exe
+  InstallerSha256: 15BC88A752F1D32AAD1242897663BBB5C502203E44545A84625FED2C75FC55C4
+  ProductCode: '{4A3949D5-8C9F-46D1-BBA6-921B56A90A23}'

```

```suggestion
  ProductCode: '{C3F0E458-1DEF-4118-89B6-B80E5A46DFE0}'
```
This needs to get updated every update.

---

It was my understanding that the product code was the only thing that connected different versions of the same product under MSI installation. If it needs to be updated, does it need to be updated to a value that comes from the installer, or to a new GUID? If it's a new GUID, what is its purpose in disambiguating a product version for which the name+version tuple is _insufficient_?

---

AFAIK most programs change the ProductCode for each version, only Appx have a consistent FamilyName which is the same between versions.

You could try PowerToys without the ProductCode since the MSI is packaged inside the exe which is causing issues and wingetcreate can't catch the ProductCode.

---

Ah, fair enough. Thanks!